lightning/src/chain/chainmonitor.rs

```diff
@@ -170,6 +171,7 @@ where
 
 	fn block_disconnected(&self, header: &BlockHeader, height: u32) {
 		let monitors = self.monitors.read().unwrap();
+		log_debug!(self.logger, "Latest block disconnected, block {} at height {} disconnected via block_disconnected", header.block_hash(), height);
```

Comment:
To remove some redundancy with the word "disconnected" and for consistency with `block_connected`, how about `"Latest block {} at height {} removed via block_disconnected"`?